### PR TITLE
Add http insecure access documentation

### PR DESCRIPTION
**Pull request by aakashnand**
@findepi @dain. I have added the documentation for `http-server.authentication.allow-insecure-over-http`. Please check if this is okay. Let me know if some changes are required.

### CODE REVIEW

File: presto-docs/src/main/sphinx/security/ldap.rst [hunk 1]
```diff
@@ -296,6 +296,10 @@ Option                          Description
 
 Troubleshooting
 ---------------
+Enable Insecure access
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+If you enable HTTPS for Presto coordinator, the default behaviour will prevent the access to web interface over HTTP with error ``Web Interface is disabled``. Although not recommended, this can be reenabled by setting
```
Comment (dain) [inline presto-docs/src/main/sphinx/security/ldap.rst:301]: I don't think you need `the` before `access`, but I do think you need it before `error`:
```
the default behaviour will prevent access to web interface over HTTP with the error ``Web Interface is disabled``
```
The most common problem is when people have an HTTPS proxy in front of the cluster and have not enabled the processing of the forwarded headers.

Comment (mosabua) [reply presto-docs/src/main/sphinx/security/ldap.rst:301]: Can this be moved to a troubleshooting section at the bottom of the web interface page instead? Its is not really related to LDAP only

Comment (mosabua) [reply presto-docs/src/main/sphinx/security/ldap.rst:301]: Beyond that I suggest to rephrase a bit avoiding future tense and reordering the sentence.

Presto disables HTTP access to the web interface when HTTPS is enabled. Although not recommended, you can enable it by setting ...

Comment (aakashnand) [reply presto-docs/src/main/sphinx/security/ldap.rst:301]: @mosabua @dain Thanks for the suggestions. I will rectify soon and will let you know for the review.

**Comment by mosabua** [comment]
This is already documented - https://trino.io/docs/current/security/tls.html#configure-the-coordinator

**Pull request closed without merging**